src/stories/drawer/Drawer.module.scss

```diff
@@ -94,7 +99,20 @@ $panelAnimationDelay: var(--pte-animations-duration-fast);
 
     &.from-right {
         right: 0;
-        padding-left: $panelMinMargin;
+
+        &.size-content, &.size-fullWithMargin {
+            padding-left: $panelMinMargin;
+        }
+        @media (width > 370px) {
```

Comment:
Okay I changed these to
`@media (min-width: 371px) { ... }`
since it seems the mid-width property translates to  `width >=` when previously I used `width > 370`, so I incremented my numbers up by one?